### ISSUE TITLE
Reolink Battery Camera

### ISSUE THREAD

**Issue by Bobsilvio** [starkillerOG/reolink_aio#93]
I read that Baichuan protocol has been implemented, but when I enter the password and port 9000 I get an error.
Tried both with the
- argus pt ultra
- atlas pt ultra

Am I doing something wrong, or is it not fully implemented yet and are we forced to use Neolink?
thanks

**Comment by starkillerOG** [comment]
Yes the Baichuan protocol has been implemented, but only the TCP part, not the UDP part.
Battery cams only use UDP, so it will not work.

Besides when keeping a constant Baichuan connection to a Reolink Battery camera I am pretty sure that will prevent the camera from sleeping. This will drain the battery. So I don't think direct connecting to the battery camera in Home Assistant will be supported anytime soon.
